Problem5/src/problem5/NewJFrame.java

```diff
@@ -0,0 +1,380 @@
+
+            /*
+            Checking the options the user chose and adding it to the variable data
+            */
+            
+            if (shooters.isSelected()) {
+                sh++;
+            }
+            if (fighters.isSelected()) {
+                fi++;
+            }
+            if (platformers.isSelected()) {
+                pl++;
+            }
+            if (strategy.isSelected()) {
+                st++;
+            }
+            if (rpg.isSelected()) {
+                rp++;
+            }
+            if (puzzle.isSelected()) {
+                pu++;
+            }
+            if (arcade.isSelected()) {
+                ar++;
+            }
+            if (sports.isSelected()) {
+                sp++;
+            }
+            if (other.isSelected()) {
+                ot++;
+            }
+            String time = hours.getSelectedItem().toString();
+            if ("< 5 hours".equals(time)) {
+                h5++;
+            }
+            if ("< 10 hours".equals(time)) {
+                h10++;
+            }
+            if ("< 15 hours".equals(time)) {
+                h15++;
+            }
+            if ("15+ hours".equals(time)) {
+                h15plus++;
+            }
+            favgames += games.getText();
+            favgames += "\n";
+            submits++;
+            
+            /*
+            Calling the method that updates the file
+            */
+            
+            updateFile();
+            
+            /*
+            Setting the program to it's starting appearance
+            */
+            
+            shooters.setSelected(false);
+            fighters.setSelected(false);
+            platformers.setSelected(false);
+            strategy.setSelected(false);
+            rpg.setSelected(false);
+            puzzle.setSelected(false);
+            arcade.setSelected(false);
+            sports.setSelected(false);
+            other.setSelected(false);
+            hours.setSelectedIndex(0);
+            games.setText("");
+
+        } catch (IOException e) {
+            System.out.println(e);
+        }
+
+
+    }//GEN-LAST:event_jButton1ActionPerformed
+
+        /**
+         *
+         * @throws IOException
+         */
+        public void updateFile() throws IOException {
+            BufferedWriter bw = null;
+            try {
+                File tempfile = new File("results.txt");
+                FileWriter fw = new FileWriter(tempfile);
+                bw = new BufferedWriter(fw);
+                PrintWriter pw = new PrintWriter(bw);
+                
+                /*
+                Using a print writer to overwrite the contents of the file 
+                and using the example given make the file look like the example
+                */
+                
+                pw.println("Number of surveys completed: " + submits);
+                pw.println("\nQuestion 1:");
+                pw.println("Shooters - " + sh + "\n"
```

Comment:
The results get reset when the program is closed.  I should be able to reopen the program and continue to add to the stats in the results.txt file.